### PR TITLE
HMA-5075: Compose SwitchRowView

### PR DESCRIPTION
**Pull request by jemmaSlater**
# 📝 Description

Adds the Material 3 SwitchRowView component built in Compose, with example usage in the sample-compose-fragments sample project.

# 🛠 Testing Notes

# 📸 Screenshots
Screenshot 2023-06-07 at 19 36 44

### CODE REVIEW

File: sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/fragment/molecules/SwitchRowViewFragment.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmrc.sample_compose_fragments.presentation.fragment.molecules
```
Comment (ngoulongkam) [inline sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/fragment/molecules/SwitchRowViewFragment.kt:1]: copyright

File: sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/molecules/SwitchRowViewScreen.kt [hunk 2]
```diff
@@ -0,0 +1,134 @@
+package uk.gov.hmrc.sample_compose_fragments.presentation.screens.molecules
```
Comment (ngoulongkam) [inline sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/molecules/SwitchRowViewScreen.kt:1]: copyright

File: sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/viewModel/SwitchRowViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,140 @@
+package uk.gov.hmrc.sample_compose_fragments.presentation.viewModel
```
Comment (ngoulongkam) [inline sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/viewModel/SwitchRowViewModel.kt:1]: copyright

**Review by ngoulongkam — CHANGES_REQUESTED** [review]

File: sample-compose-fragments/build.gradle.kts [hunk 4]
```diff
@@ -72,4 +75,12 @@ dependencies {
...
     debugImplementation("androidx.compose.ui:ui-test-manifest:${rootProject.extra["compose_version"]}")
+    constraints {
+        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.8.10") {
+            because("kotlin-stdlib-jdk7 is now a part of kotlin-stdlib")
+        }
+        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.10") {
+            because("kotlin-stdlib-jdk8 is now a part of kotlin-stdlib")
+        }
+    }
 }
```
Comment (aliahmedbd) [inline sample-compose-fragments/build.gradle.kts:86]: I have checked the Emma's PR these dependencies are in ComposeDependencies.kt and these are in build.gradle.kts; maybe there will be some conflicts.

Comment (jemmaSlater) [reply sample-compose-fragments/build.gradle.kts:86]: Thanks Ali, yeah I did copy them over from Emma's branch as I also needed to access some of the stuff from the library updates, and needed to add this to get a passing build. I think we can deal with the conflicts when they occur on either my branch or Emma's branch depending on which get's merged down first. :)

**Review by ngoulongkam — APPROVED** [review]

**Review by emmaatkins — APPROVED** [review]

**Pull request merged**